### ISSUE TITLE
alpha的设置问题

### ISSUE THREAD

**Issue by 201814476285951** [psyrocloud/MS-SSIM_L1_LOSS#6]
你好，请问损失函数中的alpha为什么要设置为0.025，《Loss Functions for Image Restoration With Neural Networks》这篇论文中好像设置为0.84，差了很多

**Comment by psyrocloud** [comment]
> 你好，请问损失函数中的alpha为什么要设置为0.025，《Loss Functions for Image Restoration With Neural Networks》这篇论文中好像设置为0.84，差了很多

你好，我的值是根据另外一个我引用的Lua代码中所取的，我并未尝试过0.84的值，您也可以按照论文中的值试试看，效果可能会有差异。这些系数的相对值会影响网络优化时的权重，若想在特定集合上取得最优的效果，好像只能大量试验和训练。
